extensions/ql-vscode/test/unit-tests/databases/db-manager.test.ts

```diff
@@ -362,6 +369,137 @@ describe("db manager", () => {
     });
   });
 
+  describe("expanded behaviours", () => {
+    it("should add item to expanded state", async () => {
+      // Add item to config
+      const listName = "my-list-1";
+      const dbConfig = createDbConfig({
+        remoteLists: [{ name: listName, repositories: [] }],
+      });
+
+      await saveDbConfig(dbConfig);
+
+      // Add item to expanded state
+      const dbItem = createVariantAnalysisUserDefinedListDbItem({
+        listName,
+      });
+
+      await dbManager.addDbItemToExpandedState(dbItem);
+      const expandedItems = await app.workspaceState.get<ExpandedDbItem[]>(
+        DbManager.DB_EXPANDED_STATE_KEY,
+      );
+
+      expect(expandedItems?.length).toEqual(1);
+      const expandedItem =
+        expandedItems![0] as VariantAnalysisUserDefinedListExpandedDbItem;
+      expect(expandedItem.listName).toEqual(listName);
+    });
+
+    it("should remove item from expanded state", async () => {
+      const listName = "my-list-2";
+      const variantAnalysisList = {
+        kind: ExpandedDbItemKind.RemoteUserDefinedList,
+        listName,
+      };
+
+      // Add item to expanded state
+      await app.workspaceState.update(DbManager.DB_EXPANDED_STATE_KEY, [
+        variantAnalysisList,
+      ]);
+      let expandedItems = await app.workspaceState.get<ExpandedDbItem[]>(
```

Comment:
Seems like we're testing our test code here (L409-L412). I don't think it's necessary.

Same comment for tests further down.

Reply:
Ok